apps/staking/src/hooks/use-stake-account.tsx

```diff
@@ -109,13 +118,16 @@ const useStakeAccountState = () => {
           signTransaction: wallet.signTransaction,
         },
       });
+      // TODO: use env var to support mainnet
+      const hermesClient = new HermesClient("https://hermes-beta.pyth.network");
```

Comment:
IMO this should be in a separate react context, it doesn't really make sense to be in this context as it's not associated to the wallet state (I should rename this hook as it's no longer just about staking accounts, it's now really a hook for the wallet state, where the list of stake accounts & the selected stake account is part of the wallet state -- and the Pyth Staking Client is here because it is only valid in certain wallet states).